app/uk/gov/hmrc/agentsubscription/model/SubscriptionRequest.scala

```diff
@@ -39,8 +39,7 @@ case class Address(addressLine1: String, addressLine2: String, addressLine3: Opt
 case class SubscriptionRequest(name: String,
                          address: Address,
```

Comment:
I'm in two minds about whether to keep the domain models for this service and for DES separate. Opinions welcome!